src/controllers/user.controller.test.ts

```diff
@@ -0,0 +1,30 @@
+import { UserController } from './user.controller'; 
+import { getUsersService } from '../services/user.services'; 
+import { UserType } from '../types/user';
+
+jest.mock('../services/user.services', () => ({
+  getUsersService: jest.fn(),
+}));
+
+const controller = new UserController();
```

Comment:
Missing testing when the endpoint fails. 
By the way, if the endpoint fails, what does it return?

Reply:
Yes. Testing and handling errors are pending.